LICENSE.TXT

```diff
@@ -0,0 +1,279 @@
+==============================================================================
```

Comment:
I'd prefer setting the `LicenseFile` property and pointing it to the original location

Reply:
Does this seem like it will work?  I'll try it out in the source-build context.